api/http/management.go

```diff
@@ -117,18 +117,16 @@ func (h *ManagementHandler) CreateIntegration(c *gin.Context) {
 
 // GET /integrations/{id}
 func (h *ManagementHandler) GetIntegrationById(c *gin.Context) {
-	var (
-		ctx = c.Request.Context()
-		id  = identity.FromContext(ctx)
-	)
-
-	if id == nil || !id.IsUser {
-		rest.RenderError(c, http.StatusForbidden, ErrMissingUserAuthentication)
+	ctx, _, err := getContextAndIdentity(c)
+	if err != nil {
 		return
 	}
 	integrationID, err := uuid.Parse(c.Param("id"))
 	if err != nil {
-		rest.RenderError(c, http.StatusNotFound, ErrIntegrationNotFound)
+		rest.RenderError(c,
+			http.StatusBadRequest,
+			errors.Wrap(err, "cannot parse path param id"),
```

Comment:
This can look pretty confusing for a consumer of the API. Why can't the path parameter be parsed? What is "path param id"?
```suggestion
			errors.Wrap(err, "integration ID must be a valid UUID"),
```

Reply:
done, thanks!